code/modules/mob/living/carbon/alien/humanoid/caste/drone.dm

```diff
@@ -20,6 +20,7 @@
 	..()
 	add_language(LANGUAGE_XENO)
 	default_language = all_languages[LANGUAGE_XENO]
+	init_language = default_language
```

Comment:
can't you move this to alien level or something

Reply:
Sure, but supercalls are all over the place. I will have to re-arrange them.